README.md

```diff
@@ -1,3 +1,13 @@
 # Hydra extensions
 
 This repository gathers vocabularies which extend [Hydra Core](http://www.hydra-cg.com/spec/latest/core/) to allow implementors to create richer hypermedia in their Hydra APIs.
+
+## Adding extensions
+
+Simply run `make new spec=my-extension` and commit the created `index.bs` file.
+
+## Run locally
+
+Run `make watch spec=my-extension` to start bikeshed in observing mode. It will generate an adjacent `index.html` when the source changes.
+
+Run `make build` to generate all specification files.
```

Comment:
I'm not familiar with this approach - how do you actually add a new extension with this?

Reply:
Look at the files. There is no magic, makefile is just a little script file to make the commands to call simple

Reply:
I can see the files. Should I type these commands in my console? What OSs are supported? How it is related to git or GitHub itself?

Reply:
> What OSs are supported?

Oh come on. Make is pretty standard anywhere and docker is required to build/watch locally but technically nothing is strictly necessary locally since the process is automated CI (if you want to give up instant feedback and wait for CI).

Speaking of, CI, preview builds could be nice for PRs so that we don't have to rely on artefacts to check the rendered result

> How it is related to git or GitHub itself?

Is it? Using GitHub Actions for CI and GitHub Pages (for now)